test/tests_on_odes/blackboxoptim_test.jl

```diff
@@ -1,20 +1,20 @@
-using BlackBoxOptim
+# using BlackBoxOptim
 
-println("Use BlackBoxOptim to fit the parameter")
-cost_function = build_loss_objective(prob1,Tsit5(),L2Loss(t,data),
-                                     maxiters=10000,verbose=false)
-bound1 = Tuple{Float64, Float64}[(1, 2)]
-result = bboptimize(cost_function;SearchRange = bound1, MaxSteps = 11e3)
-@test_broken result.archive_output.best_candidate[1] ≈ 1.5 atol=3e-1
+# println("Use BlackBoxOptim to fit the parameter")
+# cost_function = build_loss_objective(prob1,Tsit5(),L2Loss(t,data),
+#                                      maxiters=10000,verbose=false)
+# bound1 = Tuple{Float64, Float64}[(1, 2)]
+# result = bboptimize(cost_function;SearchRange = bound1, MaxSteps = 11e3)
+# @test_broken result.archive_output.best_candidate[1] ≈ 1.5 atol=3e-1
 
-cost_function = build_loss_objective(prob2,Tsit5(),L2Loss(t,data),
-                                     maxiters=10000,verbose=false)
-bound2 = Tuple{Float64, Float64}[(1, 2),(2, 4)]
-result = bboptimize(cost_function;SearchRange = bound2, MaxSteps = 11e3)
-@test_broken result.archive_output.best_candidate ≈ [1.5;3.0] atol=3e-1
+# cost_function = build_loss_objective(prob2,Tsit5(),L2Loss(t,data),
```

Comment:
`@test_broken` the blocks.

Reply:
I didn't want to import BlackBoxOptim at all. It's failing pre-compilation so seemed like a good idea to look over these completely as it stops the other errors down the line that can be fixed right now.

Reply:
Yes, comment out the import and the `@test_broken` the usage areas.